internal/cloud/run.go

```diff
@@ -310,9 +311,12 @@ func (service *runService) CancelRun(ctx context.Context, options CancelRunOptio
 }
 
 func (service *runService) GetPlanLogs(ctx context.Context, planID string) error {
+	ctxTimeout, cancel := context.WithTimeout(ctx, time.Second*10)
```

Comment:
Just nit. Putting the unit later like `10 * time.Second` seems more readable to me. Also if the same timeout is used throughout the file, maybe we can define a `const` in the file for it.

Reply:
Yeah that should be moved to a const 👍